src/Control Tasks/RockblockControlTask.hpp

```diff
@@ -39,13 +39,7 @@ class RockblockControlTask : public TimedControlTask<void>
     void dispatch_await_flush();
     void dispatch_end_transmission();
     void transition_to(rockblock_mode_type new_mode);
-    bool check_valid_command(RawRockblockCommand raw_command);
-    void check_timeout();
```

Comment:
I see these two funcs didn't exist in the cpp file - were we planning to add them or are the not necessary

Reply:
They are not necessary